act3.sh

```diff
@@ -1,13 +1,14 @@
 # 1. Unload the Millenium Falcon in ONE COMMAND!
 # Move the whole crew from the millenium_falcon directory into the death_star directory. HINT: * following a directory will grab all files/folders inside of a directory (directory/*)
-
+austinlu@Austins-MacBook-Pro death_star % mv millenium_falcon/chewbacca.txt millenium_falcon/han_solo.txt millenium_falcon/luke_skywalker.txt millenium_falcon/obi_wan.txt millenium_falcon/princess_leia.txt ~/GeneralAssembly/deliverables/Star_Wars/star_wars/empire/death_star
```

Comment:
Instead of typing out each individual file, it's possible to select all with the * like this:

`mv empire/death_star/millenium_falcon/*.txt empire/death_star`